Library/Formula/lit.rb

```diff
@@ -0,0 +1,18 @@
+class Lit < Formula
+  desc "Toolkit for developing, sharing, and running luvit/lua programs and libraries."
```

Comment:
I could be wrong but this *might* be a tad long. Limit is 80 characters, including the formula name. `brew audit --strict` would flag it if it's too long.